packages/forms-web-app/src/controllers/examination/your-interested-party-number.js

```diff
@@ -0,0 +1,71 @@
+const {
+	routesConfig: {
+		examination: {
+			pages: {
+				yourInterestedPartyNumber: {
+					view: yourInterestedPartyNumberView,
+					route: yourInterestedPartyNumberRoute
+				},
+				haveAnInterestedPartyNumber: { route: haveAnInterestedPartyNumberRoute },
+				submittingFor: { route: submittingForRoute }
+			}
+		}
+	},
+	routesConfig: { examination }
+} = require('../../routes/config');
+
+const config = require('../../config');
+
+const interestedPartyNumberString = 'interested-party-number';
+
+const examinationSession = config.sessionStorage.examination;
+const backLinkUrl = `${examination.directory + haveAnInterestedPartyNumberRoute}`;
+const pageTitle = "What's your interested party number?";
+const title = pageTitle;
+
+const getYourInterestedPartyNumber = (req, res) => {
+	const { session = { examination: { name: null } } } = req;
+	const reqExaminationSession = session[examinationSession?.name] ?? '';
+
+	if (!reqExaminationSession) return res.status(404).render('error/not-found');
+
+	const { interestedPartyNumber = '' } = reqExaminationSession;
+
+	res.render(yourInterestedPartyNumberView, {
+		backLinkUrl,
+		interestedPartyNumber,
+		pageTitle,
+		title
+	});
+};
+
+const postYourInterestedPartyNumber = (req, res) => {
+	const { body = { interestedPartyNumberString: '' } } = req;
+	const { errors = {}, errorSummary = [] } = body;
+
+	if (errors[interestedPartyNumberString] || Object.keys(errors).length > 0) {
+		res.render(yourInterestedPartyNumberView, {
+			backLinkUrl,
+			errors,
+			errorSummary,
+			pageTitle,
+			title
+		});
+		return;
+	}
+
+	req.session[examinationSession.name].property = {
```

Comment:
This would need to be `req.session[examinationSession.name][examinationSession.property. interestedPartyNumber]`